handbook/openid-connect/openid-connect.md

```diff
@@ -1,48 +1,71 @@
+You may configure SystemLink to use [OpenID Connect](https://openid.net) for user authorization. This enables SystemLink to leverage corporate single sign-on (SSO) and the additional security benefits it provides such as streamlined login and limiting the proliferation of user credentials. This also enables SystemLink to utilize a common identity for users across multiple applications. OpenID Connect can be used in conjunction with or as a replacement for LDAP, Active Directory, and local Windows accounts for authentication.
 
+## Assumptions and Prerequisites
 
+- A server running SystemLink 2020R4 or greater. Please see [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
 
-## Assumptions and Prerequisites
+- A DNS name for the SystemLink server.
+
+- SystemLink login with the **Server Administrator** role.
+
+- Administrator desktop access to the SystemLink server.
 
-- A server running SystemLink 2020R4 or greater. Please see [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server. 
-- A DNS name for the SystemLink server. 
-- SystemLink login with the **Server Administrator** role. 
-- Administrator desktop access to the SystemLink server
-- A OpenID Connect Provider server such as [PingFederate](https://www.pingidentity.com/en/software/pingfederate.html), [Azure ADFS](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs), [Okta](https://www.okta.com/openid-connect/), or another [certified provider](https://openid.net/certification/) that has been fully setup and configured for OpenID Connect authentication. 
-    - If you have not yet setup your provider please consult the vendor documentation for setup and configuration. 
+- An OpenID Connect Provider server such as [PingFederate](https://www.pingidentity.com/en/software/pingfederate.html), [Azure ADFS](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs), [Okta](https://www.okta.com/openid-connect/), or another [certified provider](https://openid.net/certification/) that has been fully setup and configured for OpenID Connect authentication.
+
+  - If you have not yet setup your provider please consult the vendor documentation for setup and configuration.
```

Comment:
It looks like mkdocs wants four spaces in order to make it a nested list. You can configure markdownlint to do this in the .markdownlint.yaml:

```yaml
MD007:
  indent: 4
```

Reply:
I went ahead and incorporated this rule.